Fix language and typos (#979)
grafana/grafana-infinity-datasource: docs/sources/examples/github.md

![image](https://user-images.githubusercontent.com/153843/93846498-1d9abf80-fc9c-11ea-90ed-4c569b088d99.png#center)

> We can leverage [Github graphql API](https://docs.github.com/en/graphql) to query the Github stats using this infinity plugin Graphql API.

Reviewer: ```suggestion
We can use the [GitHub GraphQL API](https://docs.github.com/en/graphql) to query the GitHub stats with the Infinity plugin GraphQL API.
```

## Datasource Configuration

> Select **Basic user authentication** mode and use your Github username and PAT token as password.

Reviewer: ```suggestion
Select **Basic user authentication** mode and use your GitHub username as the username and your Personal Access Token (PAT) as  thepassword.
```

![image](https://user-images.githubusercontent.com/153843/93736929-b1578780-fbd9-11ea-9413-5585ff79d3a8.png#center)

## Github Organization Summary example

![image](https://user-images.githubusercontent.com/153843/93846498-1d9abf80-fc9c-11ea-90ed-4c569b088d99.png#center)

> The query used is given below: 

Reviewer: ```suggestion
The preceding image uses the following query:
```

- Queries are not paginated
- Github rate limits apply

> - If you need a paginated and full set of results, use Grafana [Github stats plugin](https://grafana.com/grafana/plugins/grafana-github-datasource)

Reviewer: https://github.com/grafana/writers-toolkit/blob/main/vale/dictionary/g.jsonnet#L6

```suggestion
- If you need a paginated and full set of results, use Grafana [GitHub stats plugin](https://grafana.com/grafana/plugins/grafana-github-datasource)
```
